libs/agno/agno/run/workflow.py

```diff
@@ -462,6 +462,7 @@ def workflow_run_output_event_from_dict(data: dict) -> BaseWorkflowRunOutputEven
 class WorkflowRunOutput:
     """Response returned by Workflow.run() functions - kept for backwards compatibility"""
 
+    input: Optional[Union[str, Dict[str, Any], List[Any], BaseModel, Any]] = None
```

Comment:
Do we need the `Any` at the end there? Because it kinda includes everything else